.gitignore

```diff
@@ -2,3 +2,4 @@ node_modules
 yarn.lock
 .nyc_output
 umd.js
+.idea
```

Comment:
You should add this to a global gitignore instead, as most of us don't use WebStorm.

 - http://egorsmirnov.me/2015/05/04/global-gitignore-file.html
 - https://thoughtbot.com/blog/global-gitignore